languages/translations/fr.json

```diff
@@ -54,7 +54,8 @@
             "two linked by distance": "{instruction_one}, puis, dans {distance}, {instruction_two}",
             "two linked": "{instruction_one}, puis {instruction_two}",
             "one in distance": "Dans {distance}, {instruction_one}",
-            "name and ref": "{name} ({ref})"
+            "name and ref": "{name} ({ref})",
+            "exit with number": "la sortie {exit}"
```

Comment:
France accompanies exit numbers with the [![exit](https://upload.wikimedia.org/wikipedia/commons/9/95/AB-AS.svg)](https://commons.wikimedia.org/wiki/File:AB-AS.svg) icon instead of text, while Québec and New Brunswick display the number alone. But if these regions were to display text, it would omit _la_, as seen in this bilingual sign in New York:

[![](https://upload.wikimedia.org/wikipedia/commons/8/86/Northway_Exit_34_%28NY_915F%29.jpg)](https://commons.wikimedia.org/wiki/File:Northway_Exit_34_%28NY_915F%29.jpg)